encoding/ipfix/message.go

```diff
@@ -119,40 +120,50 @@ func (s TemplateSet) Type() SetType {
 
 // Header function returns Template Set's IPFIX Header.
 func (s TemplateSet) Header() SetHeader {
-	return s.SetHeader
+	return s.header
 }
 
-// TemplateRecordHeader represents a IPFIX Template Record Header.
+// TemplateRecords function returns Template Set's records.
+func (s TemplateSet) TemplateRecords() []TemplateRecord {
+	return s.records
+}
+
+// TemplateRecordHeader represents a IPFIX Template Record MessageHeader.
 type TemplateRecordHeader struct {
 	TemplateID uint16
 	FieldCount uint16
 }
 
-// A templateRecordHeaderLength defines the length of a IPFIX Template Record Header.
-// const templateRecordHeaderLength uint16 = 4
+// A templateRecordHeaderLength defines the length of a IPFIX Template Record MessageHeader.
+const templateRecordHeaderLength uint16 = 4
 
 // A TemplateRecord represents a IPFIX Template Record.
 // https://www.ietf.org/rfc/rfc7011.html#section-3.4.1
 type TemplateRecord struct {
-	// Header          TemplateRecordHeader
+	TemplateHeader  TemplateRecordHeader
 	FieldSpecifiers []FieldSpecifier
 }
 
 // A DataSet type represents a IPFIX Data Set.  It consists of a SetHeader and number of DataRecords.
 // https://tools.ietf.org/html/rfc7011#section-3.4.3
 type DataSet struct {
-	SetHeader SetHeader
-	Records   []DataRecord
+	header  SetHeader
+	records []DataRecord
 }
 
 // Type function returns DataSetType for any DataSet.
 func (s DataSet) Type() SetType {
 	return DataSetType
 }
 
-// Header function returns Data Set's IPFIX Header.
+// MessageHeader function returns Data Set's IPFIX MessageHeader.
```

Comment:
Codacy found an issue: [comment on exported method DataSet.Header should be of the form "Header ..."](https://app.codacy.com/manual/mateumann/clouseau/pullRequest?prid=6326016)